clang/test/SemaSYCL/reqd-sub-group-size-ast.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -ast-dump %s | FileCheck %s
+// RUN: %clang_cc1 -fsycl-is-device -Wno-error=deprecated-declarations -internal-isystem %S/Inputs -ast-dump %s | FileCheck %s
 
 // The test checks AST of [[intel::reqd_sub_group_size()]] attribute.
```

Comment:
This is likely a legacy from times when `sycl::reqd_sub_group_size` didn't even exist. I suggest to do a renaming here as well